ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20-periodics.yaml

```diff
@@ -1,23 +1,24 @@
 periodics:
 - agent: kubernetes
   cluster: build05
-  cron: 0 0 * * *
+  cron: 0 4 * * *
   decorate: true
   decoration_config:
     skip_cloning: true
+    timeout: 5h0m0s
   extra_refs:
   - base_ref: release-4.20
     org: openshift
     repo: ovn-kubernetes
   labels:
-    ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/cloud: equinix-ocp-metal
+    ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
     ci-operator.openshift.io/cluster: build05
     ci-operator.openshift.io/variant: periodics
     ci.openshift.io/generator: prowgen
     job-release: "4.20"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-ovn-kubernetes-release-4.20-periodics-e2e-aws-kubevirt-ovn-techpreview
+  name: periodic-ci-openshift-ovn-kubernetes-release-4.20-periodics-e2e-metal-ipi-ovn-dualstack-bgp-local-gw-techpreview
```

Comment:
huh why are we replacing lgw bgp jobs? i'd hope these jobs graduate to GA when @jcaamano  gets BGP to GA